core/models.py

```diff
@@ -528,10 +534,12 @@ def is_active(self):
         return True
 
     def has_perm(self, perm, obj=None):
-        i_user = self.i_user if obj is None else obj.i_user
+        self.has_perms(self, perm, obj=None)
+
+    def has_perms(self, perm, obj=None):
```

Comment:
`x if y else z` is good syntactic sugar when you have one liners in this case please change implementation to regular if else statement

Reply:
It is artifact from reversing those changes, this code is not changed in comparison to previous version:
https://github.com/openimis/openimis-be-core_py/blob/main/core/models.py#L530